helper/kv-builder/builder.go

```diff
@@ -48,33 +48,36 @@ func (b *Builder) add(raw string) error {
 		return nil
 	}
 
+	// Split into key/value
+	parts := strings.SplitN(raw, "=", 2)
+
 	// If the arg is exactly "-", then we need to read from stdin
 	// and merge the results into the resulting structure.
-	if raw == "-" {
-		if b.Stdin == nil {
-			return fmt.Errorf("stdin is not supported")
-		}
-		if b.stdin {
-			return fmt.Errorf("stdin already consumed")
+	if len(parts) == 1 {
```

Comment:
I think this would break `foo=@policy.hcl` and `foo=-`. it should be checking againsts `parts[1][0]`